### DISCUSSION TITLE
How about implement portrait mode in osu-mania for mobile ?

### DISCUSSION THREAD

**Discussion by gaikoz11** [ppy/osu#21605] [Ideas]
Im sure its impossible but for mobile platform IOS or ANDOIR portrait mode make more usable Because of the screen-size   example like piano tile style ? . Or like this one i cut in portrait mode style
![Untitled14_20221211191350](https://user-images.githubusercontent.com/51250782/206900491-d53cd010-32e5-4e8b-b3a9-98cf5a88503d.png)

**Comment by peppy** [comment]
https://github.com/search?q=repo%3Appy%2Fosu+portrait&type=discussions
